### PR TITLE
Replace game.Lighting with game:GetService("Lighting")

### PR DESCRIPTION
**Pull request by Katrist**
## Changes

Added a Lighting variable for every page that referenced game.Lighting, and replaced all instances of game.Lighting with Lighting.

## Checks

By submitting your pull request for review, you agree to the following:

- [X] This contribution was created in whole or in part by me, and I have the right to submit it under the terms of this repository's open source licenses.
- [X] I understand and agree that this contribution and a record of it are public, maintained indefinitely, and may be redistributed under the terms of this repository's open source licenses.
- [X] To the best of my knowledge, all proposed changes are accurate.

### CODE REVIEW

File: content/en-us/players/tools.md [hunk 1]
```diff
@@ -169,12 +169,14 @@ The following example shows steps for adding a `Class.Script` on the server that
 2. Copy the following code and paste it into your `Class.Script`.
 
    ```lua
+   local Lighting = game:GetService("Lighting")
+
    local tool = script.Parent
    local function onActivate()
-    if game.Lighting.ClockTime \>= 8 and game.Lighting.ClockTime < 16 then
-      game.Lighting.ClockTime = 20
+    if Lighting.ClockTime \>= 8 and Lighting.ClockTime < 16 then
```
Comment (colbert2677) [inline content/en-us/players/tools.md:176]: Stray backslash invalidates the first operator (\>= is not a valid).

*Note: this is out of scope of the PR, as this is not related to changing dot indexing services to use GetService. I've therefore left this as a comment and not a request for change.*

Comment (Katrist) [reply content/en-us/players/tools.md:176]: I sent a PR to change that earlier but forgot to change it back when the PR was merged, thanks for notifying me.

**Review by IgnisRBX — APPROVED** [review]
